packages/docusaurus-plugin-content-docs/src/sidebars.ts

```diff
@@ -127,7 +127,10 @@ function assertIsCategory(
     );
   }
   // "collapsed" is an optional property
-  if (item.hasOwnProperty('collapsed') && typeof item.collapsed !== 'boolean') {
+  if (
+    Object.prototype.hasOwnProperty.call(item, 'collapsed') &&
```

Comment:
too much ceremony, I'd just replace by `typeof item.collapsed !== "undefined"` ?

Reply:
There is actually a consideration about `collapsed` existing on the prototype chain of `item`. But I think it's safe to assume here that `typeof item.collapsed !== "undefined"` works the same...

Reply:
yes it looks to me as the sidebar is a regular object

Reply:
can you replace with `typeof item.collapsed !== "undefined"` ? it has not been done here